io/include/pcl/io/impl/synchronized_queue.hpp

```diff
@@ -38,10 +38,10 @@
 /* SynchronizedQueue Template, taken from
  * http://stackoverflow.com/questions/10139251/shared-queue-c
  */
+#pragma once
```

Comment:
I'm not really sure why I made this to the if guards. I can revert it later if desired.

Reply:
You probably realized that the current one is not correct (`H` vs `HPP`), leave alone the fact that we were supposed to switch to `#pragma once` anyway.